frontend/src/routes/__root.tsx

```diff
@@ -4,10 +4,10 @@ import { TailwindIndicator } from "~/components/tailwind-indicator";
 
 export const Route = createRootRoute({
   component: () => (
-    <>
+    <div className="bg-background">
```

Comment:
```suggestion
    <main className="bg-background">
```

Use `main` it is more semantic